recipe/meta.yaml

```diff
@@ -22,11 +22,12 @@ requirements:
     - python >=3.7
     - google-api-core-grpc >=1.32.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*
     - grpc-google-iam-v1 >=0.12.4,<1.0.0dev
-    - libcst >=0.3.10
-    - proto-plus >=1.15.0,<2.0.0dev
-    - protobuf >=3.19.0,<4.0.0dev
     - grpcio >=1.38.1,<2.0dev
     - grpcio-status >=1.16.0
+    - proto-plus >=1.22.0,<2.0.0dev
+    - protobuf >=3.19.0,<5.0.0dev
+    # from libcst extra
+    - libcst >=0.3.10
```

Comment:
Just reordering packages to be more like what grayskull generates.  This tends to make mantenance easier in my experience.  Happy to change back if needed but the order here is neither grayskull's nor the upstream package's.

Reply:
Whatever you prefer.